### ISSUE TITLE
Alias doesn't catch library

### ISSUE THREAD

**Issue by morags** [freeCodeCamp/devdocs#925]
Problem:
The "py" alias isn't catching a core library

Steps to reproduce:
1. Type `py pprint` into the search box: search lists the `pprint` core library
2. Type `py json` into the search box: search does not list the `json` core library
3. Type `json` alone: Python's `json` library is listed

Attempted resolution:
1. Deleted browser caches and reloaded the page, but the problem recurs

**Comment by jmerle** [comment]
This sounds like the intended behavior. When you search for something, DevDocs will look for any page that (somewhat) matches it. An alias, however, means that you can type "py", press tab and then continue searching in _only_ the selected documentation.

Try it for yourself: type "py", press <kbd>tab</kbd> and type "json":

![](https://i.imgur.com/sKAvruD.gif)

**Comment by j-f1** [comment]
Just tested this too. I can’t reproduce it either. Which Python version(s) do you have enabled @morags?

**Comment by morags** [comment]
Python 3.7.

If that's the behavior of aliases then it needs to be clarified in the docs. I took it as something like Firefox's "quick search" (or Sublime's / VS Code's commands), where the text is filtered (or matched) by simply typing, without needing to auto-complete a filter with `TAB`. For example, if I type `pydoc pprint` into my browser, it automatically launches a [query](https://docs.python.org/3/search.html?q=pprint&check_keywords=yes&area=default) to Python's docs. I do not need to specify that "pydoc" is a keyword (or filter) when I use it, because I've already defined it as such earlier.

Bottom line: I suggest either clarifying this behavior in the docs ("aliases are used in the following way...") or changing the aliases' behavior so they're always used, regardless of auto-completion.
